### DISCUSSION TITLE
Форматирование названий папок дерева

### DISCUSSION THREAD

**Discussion by khpavel** [totumonline/totum-ru-issues-and-discussions#507] [Как это сделать?] [answered]
Есть задача форматирования названий папок дерева (поле типа Селект-дерево). 
В настройках есть отдельное поле "Код форматирования папок дерева". Но как его использовать непонятно. Раздел справки пустой. Функция SetFormat не работает.

Можно где-то найти описание?

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Оно не подключено и в ближайшие не знаю сколько лет не подключится.
